src/Console/Output/ConsoleOutputFormatter.php

```diff
@@ -37,7 +38,7 @@ public function __construct(SymfonyStyle $symfonyStyle, BetterStandardPrinter $b
         $this->betterStandardPrinter = $betterStandardPrinter;
     }
 
-    public function report(ErrorAndDiffCollector $errorAndDiffCollector): void
+    public function report(ErrorAndDiffCollector $errorAndDiffCollector, Configuration $configuration): void
```

Comment:
Why passing service as a method argument?
Should be in constructor if needed.

Reply:
Oh right, it is a service indeed. At first I thought it was a value object. I'll change it.

Reply:
I see. Could you think of better name for it so it's recognizable?

Reply:
I don't think the name is wrong or misleading. Those were just my expectations of Symfony Command input arguments handling. ;)